### PR TITLE
ceis lookup implementation

### PR DESCRIPTION
**Pull request by TayGov**
# Description

This PR includes the following proposed change(s):

Add ords swagger
Implement generated code
Add mapper
Return mapped values 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Unit tests

## Does the change impact or break the Docker build?

- [ ] Yes
- [X] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [X] No

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/backend/libs/efiling-commons/src/main/java/ca/bc/gov/open/jag/efilingcommons/utils/BooleanUtils.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package ca.bc.gov.open.jag.efilingcommons.utils;
+
+import org.springframework.util.StringUtils;
+
+public class BooleanUtils {
+
+    private BooleanUtils() {}
+
+    public static Boolean toBoolean(String value) {
+        if (StringUtils.isEmpty(value)) return null;
```
[the remark this answers is not in the document]
Comment (TayGov) [reply src/backend/libs/efiling-commons/src/main/java/ca/bc/gov/open/jag/efilingcommons/utils/BooleanUtils.java:10]: This is tri state

File: src/backend/libs/efiling-ceis-api-client/src/test/java/ca/bc/gov/ca/open/jag/ceis/AutoConfigurationTest.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
+package ca.bc.gov.ca.open.jag.ceis;
```
[the remark this answers is not in the document]
Comment (TayGov) [reply src/backend/libs/efiling-ceis-api-client/src/test/java/ca/bc/gov/ca/open/jag/ceis/AutoConfigurationTest.java:1]: This test looks like others

File: src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/courts/CeisLookupAdapter.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+public class CeisLookupAdapter {
+
+    private final DefaultApi defaultApi;
+
+    public CeisLookupAdapter(DefaultApi defaultApi) {
+        this.defaultApi = defaultApi;
+    }
+
+    public CourtLocations getCourLocations(String courtType) throws ApiException {
```
[the remark this answers is not in the document]
Comment (TayGov) [reply src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/courts/CeisLookupAdapter.java:27]: This is used ?

File: src/backend/efiling-api/src/test/java/ca/bc/gov/open/jag/efilingapi/courts/GetCourtLocationsTest.java [hunk 4]
```diff
@@ -54,4 +71,50 @@ public void withValidCourtNameReturnCourtLocations() {
...
+
+        ResponseEntity actual = sut.getCourtLocations(COURTLEVELERROR);
+
+        Assertions.assertEquals(HttpStatus.INTERNAL_SERVER_ERROR, actual.getStatusCode());
+        Assertions.assertEquals(COURT_LOCATION_ERROR.getErrorCode(), ((EfilingError)actual.getBody()).getError());
+        Assertions.assertEquals(COURT_LOCATION_ERROR.getErrorMessage(), ((EfilingError)actual.getBody()).getMessage());
+        
+    }
+
+    private CourtLocations buildMockData() {
```
[the remark this answers is not in the document]
Comment (TayGov) [reply src/backend/efiling-api/src/test/java/ca/bc/gov/open/jag/efilingapi/courts/GetCourtLocationsTest.java:87]: Test class

File: src/backend/libs/efiling-commons/src/main/java/ca/bc/gov/open/jag/efilingcommons/utils/BooleanUtils.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (akroon3r) [reply src/backend/libs/efiling-commons/src/main/java/ca/bc/gov/open/jag/efilingcommons/utils/BooleanUtils.java:10]: we're using super position?

File: src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/courts/CeisLookupAdapter.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (akroon3r) [reply src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/courts/CeisLookupAdapter.java:27]: definitely used in the filter, I wonder what happens if you pass the parameter into a variable before using it in the filter...

File: src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/courts/CourtsApiDelegateImpl.java [hunk 5]
```diff
@@ -1,54 +1,40 @@
...
+
+    public CourtsApiDelegateImpl(CeisLookupAdapter ceisLookupAdapter) {
+        this.ceisLookupAdapter = ceisLookupAdapter;
+    }
+
     @Override
     public ResponseEntity<CourtLocations> getCourtLocations(String courtLevel) {
-        return ResponseEntity.ok(buildMockData());
+        try {
+            logger.info("Request for court level recieved {}", courtLevel);
```
Comment (akroon3r) [inline src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/courts/CourtsApiDelegateImpl.java:29]: is the a uid passed/generated through the request that we can log here? sorry if dumb question.

**Review by akroon3r — APPROVED** [review]
I think this is fine, maybe needs Alex for a second set of eyes.

**Review by alexjoybc — APPROVED** [review]

**Pull request merged**